style.css

```diff
@@ -235,6 +250,17 @@
   height: auto;
 }
 
+.dec {
+  display: none;
+}
+
+.canopy {
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  color: #344563;
+}
+
 .card-content {
```

Comment:
- Kindly, your desktop cards need some space to stick the Figma design 👍😁, kindly update based n the [given design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848&t=VJop6PK5cSpPL27i-0)👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/213581616-b2632b9d-eaeb-44a6-a542-7b49bbcde9f5.png" width="600">|<img src="https://user-images.githubusercontent.com/52010789/213581624-5db1f9b6-27f5-4b42-9700-e8ac466c0771.png" width="600">|


😎️
🤝️